moveit_ros/planning/planning_scene_monitor/src/planning_scene_monitor.cpp

```diff
@@ -51,6 +51,8 @@ namespace planning_scene_monitor
 
 using namespace moveit_ros_planning;
 
+static const std::string logger_name = "planning_scene_monitor";
```

Comment:
Should be [all caps](http://wiki.ros.org/CppStyleGuide#Constants) since its a constant

Also, I used the var to keep the line length short - can we have a shorter var name? I had it as ``name_``, but if you don't like that how about ``CNAME``?